### PR TITLE
Feat components

### PR DESCRIPTION
**Pull request by humzahasan**
PR Description :

PR contains changes made for common components added to be used throughout application.

No Preview available since only components are added here!

[Check Me Only](https://github.com/humzahasan/notova/pull/4/commits/ed81c8d0aec1a2da8fb431551f8e7bb3bdad59da)

### CODE REVIEW

File: src/Components/Sidebar/Sidebar.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  width: 100%;
+  margin: 0;
+  padding: 1rem;
+  text-decoration: none;
+  color: inherit;
+}
+
+.sidebar-account {
+  display: flex;
+  /* width: 50%; */
```
Comment (zoyatanweer) [inline src/Components/Sidebar/Sidebar.css:26]: remove commented code

File: src/Components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -1,36 +1,79 @@
...
+  let activeStyle = {
+    backgroundColor: "var(--primary-color-light)",
+    fontWeight: "500",
+  };
+  const { getUser } = useAuth();
+  const getFullName = `${getUser().firstName} ${getUser().lastName}`;
+  const avatarName = `${getUser().firstName[0]}${getUser().lastName[0]}`;
+
+  const logoutHandler = () => {
+    console.info("User Logged out!");
```
Comment (archit7448) [inline src/Components/Sidebar/Sidebar.jsx:16]: you can remove console.info

**Review by zoyatanweer — APPROVED** [review]
- very clean code, easily understandable
- loved the way you wrote comments to tell the user what it is
- good to merge

**Review by archit7448 — APPROVED** [review]
- you can add image or netlify link for getting sense of ui 
- code is clean 
- looks good to merge

File: src/Components/Sidebar/Sidebar.css (hunk 1 above)
Comment (humzahasan) [reply src/Components/Sidebar/Sidebar.css:26]: Will do!

File: src/Components/Sidebar/Sidebar.jsx (hunk 2 above)
Comment (humzahasan) [reply src/Components/Sidebar/Sidebar.jsx:16]: Will do !

**Pull request merged**